observer.py

```diff
@@ -33,8 +34,19 @@ def get_publishers(network):
     return json_data.get(network, {})
 
 
+def filter_errors(regexes, errors):
+    filtered_errors = []
+    for e in errors:
+        skip = False
+        for r in regexes:
+            if re.search(r, f"{e.symbol}/{e.error_code}"):
```

Comment:
I think you might want to use `re.match` here and force the user to put `.*` in the regex where they expect wildcard matches. I think that's a little more intuitive than having them put in `^` and `$` to anchor the string.

Also, maybe this should be a case-insensitive match ?